br.com.zoop.zooppcserver.yml

```diff
@@ -0,0 +1,37 @@
+app-id: br.com.zoop.zooppcserver
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: launcher
+finish-args:
+  - --share=network
+  - --device=dri
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --env=PATH=/app/bin:/usr/bin:/app/jre/bin
+  - --env=JAVA_HOME=/app/jre
+modules:
+  - name: ZoopPcServer
+    buildsystem: simple
+    build-commands:
+      - install -d ${FLATPAK_DEST}/bin
+      - cp launcher ${FLATPAK_DEST}/bin/
+      - cp pinpad-server.jar ${FLATPAK_DEST}/bin/
+      - chmod +x ${FLATPAK_DEST}/bin/launcher
+      - install -d ${FLATPAK_DEST}/share/applications
+      - install -Dm644 br.com.zoop.zooppcserver.desktop -t ${FLATPAK_DEST}/share/applications/
+      - install -d ${FLATPAK_DEST}/share/icons/hicolor/256x256/apps
+      - install -Dm644 br.com.zoop.zooppcserver.png -t ${FLATPAK_DEST}/share/icons/hicolor/256x256/apps/
```

Comment:
the image is not 256x256.... 
its size is actually non standard and smaller than the minimum 128x128